libs/libxml2/v2.10.3/wl-build.sh

```diff
@@ -26,18 +26,17 @@ cd "${WASMLABS_SOURCE_PATH}"
 
 if [[ -z "$WASMLABS_SKIP_CONFIGURE" ]]; then
     ./autogen.sh
-    export LIBXML2_CONFIGURE='--enable-static --disable-shared --with-minimum=yes --with-output=yes --with-schemas=yes --with-tree=yes --with-valid=yes --with-html=yes --with-xpath=yes --with-reader=yes --with-writer=yes --with-xinclude=yes --with-c14n=yes --with-sax1=yes'
-    logStatus "Configuring build with '${LIBXML2_CONFIGURE}'... "
-    ./configure --host=wasm32-wasi host_alias=wasm32-musl-wasi --target=wasm32-wasi target_alias=wasm32-musl-wasi ${LIBXML2_CONFIGURE} || exit 1
+    export LIBXML2_CONFIGURE="--prefix="${WASMLABS_OUTPUT}" --enable-static --disable-shared --with-minimum=yes --with-output=yes --with-schemas=yes --with-tree=yes --with-valid=yes --with-html=yes --with-xpath=yes --with-reader=yes --with-writer=yes --with-xinclude=yes --with-c14n=yes --with-sax1=yes"
```

Comment:
Is this double quote within double quotes working as expected?, shouldn't we escape the inner ones?

Reply:
It's a concatenation actually, leaving outside the quotes the variables substitutions.